### PR TITLE
[DO NOT MERGE] CAPT-1155, CAPT-1093  New induction question and admin task checking with DQT

### PR DESCRIPTION
**Pull request by your**
Merging the two branches into one, we can use a review app to test user and admin-facing functionality in one go.

From an implementation point of view, both tickets are independent and can be reviewed and released separately. This is possible because we are not validating the answer provided by the claimant unless it's an ECP claim and it's been indicated that induction has been completed (which means the claim has been submitted). We won't ever accept an ECP claim unless induction has been indicated as completed; the "you may be eligible later" screen is displayed instead and without further questions. 

The question is shown at the beginning of the ECP/LUP journey, but it's actually irrelevant if the claimant is eventually eligible for LUP only. In this case, the admin task is not applicable.  

~~If merged, this PR replaces and closes #2420 and #2425.~~ The most likely scenario is that both individual PRs will supersede this PR.

### CODE REVIEW

File: app/views/early_career_payments/claims/eligible_later.html.erb [hunk 1]
```diff
@@ -6,28 +6,36 @@
...
+      <% end %>
 
-    <% if current_claim.eligibility.postgraduate_itt? %>
-      <p class="govuk-body">
-        You are not eligible for the
-        <%= link_to "early-career payment (opens in new tab)", "https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools", class: "govuk-link govuk-link--no-visited-state", target: "_blank" %>
-        because you started your postgraduate initial teacher training in the <%= current_claim.eligibility.itt_academic_year.to_s(:long) -%> academic year.
-      </p>
-    <% end %>
+      <% if current_claim.eligibility.postgraduate_itt? %>
```
Comment (martin-dfe) [inline app/views/early_career_payments/claims/eligible_later.html.erb:24]: Hey there, can you remove extra spaces.

File: app/views/early_career_payments/claims/induction_completed.html.erb [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<% content_for(:page_title, page_title(t("early_career_payments.questions.induction_completed.heading"), policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
+<% path_for_form = current_claim.persisted? ? claim_path(current_policy_routing_name) : claims_path(current_policy_routing_name) %>
+<% shared_view_css_size = current_claim.policy == EarlyCareerPayments ? "l" : "xl" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { "eligibility.induction_completed": "claim_eligibility_attributes_induction_completed_true" }) if current_claim.errors.any? %>
```
Comment (martin-dfe) [inline app/views/early_career_payments/claims/induction_completed.html.erb:7]: Is this a GitHub view or the line is actually breaking into two lines?

File: app/views/early_career_payments/claims/eligible_later.html.erb [hunk 3]
```diff
@@ -6,28 +6,36 @@
...
+        <p class="govuk-body">
```
Comment (martin-dfe) [inline app/views/early_career_payments/claims/eligible_later.html.erb:25]: hey there, can you please remove the extra blank spaces?

File: app/views/early_career_payments/claims/induction_completed.html.erb (hunk 2 above)
Comment (your) [reply app/views/early_career_payments/claims/induction_completed.html.erb:7]: @martin-dfe this `render` statement is on one line; pretty standard, I copied it from other similar views.

**Comment by your** [comment]
Superseded by https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2420 and https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2425

**Pull request closed without merging**